vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -2,570 +2,264 @@
+|`commands`|Command[]|No||Custom commands definitions|
+
+#### Microservices Sample Code
+```json
+"microservices": [
+    {
+      "name": "my-ms",
+      "stack": "spring-boot",
+      "dbms": "mysql",
+      "ingressPath": "/ingress",
+      "healthCheckPath": "/management/health",
+      "roles": ["admin"],
+      "env": [{ "VAR_1": "value" }]
+    }
+  ],
+```
+
+### Micro Frontends Specifications
+|Name|Type|Required|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|Yes||Micro frontend name|
+|`stack`|Enum|Yes|*react   *angular|MFE stack|
+|`type`|Enum|Yes|*widget  *widget-config  *app-builder|Type of MFE|
+|`slot`|Enum|Yes for `type=app-builder`|*primary-header  *primary-menu  *content|Named reference to an App Builder embedded position in a specific layout|
+|`paths`|String[]|Yes for `type=app-builder` and `slot=content`||App Builder activation paths|
+|`titles`|String[]|Yes for `type=widget`||Localized widget labels|
+|`group`|String|Yes||Visibility group name|
+|`publicFolder`|String|No|Default is `public`|MFE public folder (typically where index.html is located)|
+|`apiClaims`|String[]|No||See [API Claim spec](#api-claim-specification) below|
+|`nav`|[MenuEntry[]](#menuentry-specification)|No||Bundle menu global links|
+|`commands`|Command[]|No||Custom commands definitions|
+|`buildFolder`|String|No|Default is `build`|Corresponds to the MFE build folder |
+|`mfeConfig`| MfeConfig[]|No||Custom configuration widget for App Builder|
+
+#### Micro Frontends Sample Code 
+```json
+ "microfrontends": [
+    {
+      "name": "my-mfe",
+      "stack": "react",
+      "titles": { "en": "my_en_mfe_title", "it": "my_it_mfe_title" }
+      "type": "app-builder",
+      "slot": "content",
+      "paths": ["/path1"],
+      "group" : "free",
+      "apiClaims": [...],
+    },
+    {
+      "name": "my-mfe2",
+      "stack": "react",
+      "type": "widget",
+      "publicFolder": "public",
+      "titles": { "en": "my_en_mfe2_title", "it": "my_it_mfe2_title" },
+      "group": "free",
+      "commands": { "build": "custom-command" }
+    }
+```
+#### mfeConfig Specifications
+|Name|Type|Required|Description|
+|:-|:-|:-|:------------------------|
+|params| MfeParam[]  |Yes| User configuration for executing a widget|
+|contextParams|String[]| Yes | Information extracted from the application context |
+|systemParams|String[]| Yes | Static and system settings for execution of widget |
+
+#### MfeParam Specification
+|Name|Type|Required|Description|
+|:-|:-|:-|:------------------------|
+|name|String|Yes|Name of the parameter|
+|description|String|No|Description of the parameter|
+
+```json
+{
+  "systemParams": {
+    "api": {
+      "int-api": {
+        "url": "apiClaim_int__DASH__api"
+      },
+      "ext-api": {
+        "url": "apiClaim_ext__DASH__api"
+      }
+    }
+  },
+  "contextParams": {
+    "page_code": "${page_code}",
```

Comment:
I think we need to remove all of the ${page_code] bits here in lines 186-188. I think that was pseudo-code used by Engineering to indicate their implementation. The actual working code (although I'm setting up to test it) doesn't include values. See https://github.com/entando/entando-showcase-bundle/blob/master/entando.json for a real example.